cmd/mailchain/internal/http/handlers/balance.go

```diff
@@ -0,0 +1,157 @@
+// Copyright 2019 Finobo
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package handlers
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/gorilla/mux"
+	"github.com/mailchain/mailchain/cmd/internal/http/params"
+	"github.com/mailchain/mailchain/errs"
+	"github.com/mailchain/mailchain/internal/address"
+	"github.com/mailchain/mailchain/internal/mailbox"
+	"github.com/pkg/errors"
+)
+
+// GetBalance returns the balance of a user.
+func GetBalance(balanceFinder map[string]mailbox.BalanceFinder) func(w http.ResponseWriter, r *http.Request) {
+	// Get swagger:route GET /balance Balance GetBalance
+	//
+	// Get balance.
+	//
+	// Get the  balance of the user. The balance is used to send or receive messages.
+	// Responses:
+	//   200: GetBalanceResponse
+	//   404: NotFoundError
+	//   422: ValidationError
+	return func(w http.ResponseWriter, r *http.Request) {
+		ctx := r.Context()
+		req, err := parseGetBalance(r)
+		if err != nil {
+			errs.JSONWriter(w, r, http.StatusUnprocessableEntity, errors.WithStack(err))
+			return
+		}
+		fmt.Println(balanceFinder)
```

Comment:
`Println` can be good for WIP but should be removed for the release. If you think there is something valuable to log take a look at the usage of `"github.com/rs/zerolog/log"` within the codebase.

Reply:
Thank you, Println is now removed.